Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabSystemComponent.cpp

```diff
@@ -810,34 +810,8 @@ namespace AzToolsFramework
             PrefabDom& targetTemplateDom = targetTemplate.GetPrefabDom();
             auto memberFound = targetTemplateDom.FindMember(PrefabDomUtils::InstancesName);
 
-            PrefabDomValueReference instancesValue;
-            if (memberFound == targetTemplateDom.MemberEnd())
+            if (PrefabDomUtils::AddOrUpdateNestedInstance(targetTemplateDom, instanceAlias))
```

Comment:
Curious why a nullopt nestedInstance is used here

Reply:
That call is in CreateLink() which updates the template further down in newLink.UpdateTarget().